### PR TITLE
Add Angular to website 🎉 

### PR DESCRIPTION
**Pull request by alisonjoseph**
**Staging link ->[ design-system-angular.mybluemix.net](design-system-angular.mybluemix.net)**

Adds angular docs to
- Homepage
- [Getting Started](http://design-system-angular.mybluemix.net/getting-started/developers/angular)
- [contributing/developers](http://design-system-angular.mybluemix.net/getting-started/contributing/developers)

Updates left nav to two buttons, links to [new section on resources page with github repos](http://design-system-angular.mybluemix.net/resources#github)

Adds Angular link to component pages where applicable 

Updated [component status](http://design-system-angular.mybluemix.net/component-status) page

### CODE REVIEW

**Comment by alisonjoseph** [comment]
FYI @cal-smith @zvonimirfras

File: src/content/getting-started/developers/other.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-**The Carbon team currently only officially supports vanilla js and React. However developers using a different javascript framework (such as Angular or Vue) can can follow the instructions for the [Vanilla](/getting-started/developers/vanilla) library to access the styles and build out their own components.**
+**The Carbon team currently supports vanilla js, React and Angular. Developers wanting to use another frameworkyar can can follow the instructions for the [Vanilla](/getting-started/developers/vanilla) library to access the styles and build out their own components.**
```
Comment (joshblack) [inline src/content/getting-started/developers/other.md]:
```diff
-The Carbon team currently supports vanilla js, React and Angular. Developers wanting to use another frameworkyar can can
+The Carbon team currently supports vanilla js, React and Angular. Developers wanting to use another framework can

Comment (alisonjoseph) [reply src/content/getting-started/developers/other.md]: "frameworkyar" lol

File: src/app/components/internal/ComponentExample/ComponentExample.js [hunk 2]
```diff
@@ -164,6 +182,34 @@ class ComponentExample extends Component {
...
     const counter = Math.floor(Math.random() * 100) + 1;
+    const reactLink = `http://react.carbondesignsystem.com/?selectedKind=${componentNameLink}`;
+    
+    
+
+    if (
+      component === 'checkbox' ||
+      component === 'label'
+    ) {
+      var angularLink = `http://angular.carbondesignsystem.com/?selectedKind=Forms&selectedStory=${componentNameLink}`;
```
Comment (asudoh) [inline src/app/components/internal/ComponentExample/ComponentExample.js]: Great to have this tabular-ized, too, like:

```javascript
const angularLink = {
  ...
  notification: `http://angular.carbondesignsystem.com/?selectedKind=Banner`,
  toggle: `http://angular.carbondesignsystem.com/?selectedKind=Forms&selectedStory=switch`,
  ...
}[component];
```

File: src/app/components/internal/ComponentExample/ComponentExample.js [hunk 3]
```diff
@@ -138,8 +138,26 @@ class ComponentExample extends Component {
       componentNameLink = 'Progress%20Indicator';
     }
     let hasReactVersion = true;
+    let hasAngularVersion = false;
     let hasLightVersion = false;
 
+    if (
+      component === 'accordion' ||
```
[the remark this answers is not in the document]
Comment (alisonjoseph) [reply src/app/components/internal/ComponentExample/ComponentExample.js]: I'm going to leave this as is for now, but in the future (gatsby site) can hopefully use/update the component-status.json file so we can use it across the site for this?

File: src/app/components/internal/ComponentExample/ComponentExample.js (hunk 2 above)
Comment (alisonjoseph) [reply src/app/components/internal/ComponentExample/ComponentExample.js]: There is a default link so I wasn't able to use what you suggested here, but I did update the js to use switch. Seems a bit cleaner now?

Comment (asudoh) [reply src/app/components/internal/ComponentExample/ComponentExample.js]: @alisonjoseph switch sounds OK - For the record, the “default” scenario can be covered by:

```javascript
{
  foo: fooValue,
  bar: barValue,
][key] || defaultValue;
```

Comment (alisonjoseph) [reply src/app/components/internal/ComponentExample/ComponentExample.js]: 👍 thanks Akira! I didn't realize 😄

**Pull request merged**